src/Docfx.Build.Common/ModelAttributeHandlers/HandleModelAttributesContext.cs

```diff
@@ -8,17 +8,17 @@ namespace Docfx.Build.Common;
 
 public class HandleModelAttributesContext
 {
-    internal int NestedLevel { get; set; } = 0;
-    public IHostService Host { get; set; }
-    public bool SkipMarkup { get; set; }
-    public bool EnableContentPlaceholder { get; set; }
-    public string PlaceholderContent { get; set; }
-    public bool ContainsPlaceholder { get; set; }
-    public HashSet<string> Dependency { get; set; } = new HashSet<string>();
-    public FileAndType FileAndType { get; set; }
-    public HashSet<string> LinkToFiles { get; set; } = new HashSet<string>(FilePathComparer.OSPlatformSensitiveStringComparer);
-    public HashSet<string> LinkToUids { get; set; } = new HashSet<string>();
-    public List<UidDefinition> Uids { get; set; } = new List<UidDefinition>();
+    internal int                 NestedLevel              { get; set; } = 0;
```

Comment:
Consider disable adding spaces to align property names. The rest of the codebase does not follow this style pattern.